pkgs/development/python-modules/dashing/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, python3
```

Comment:
Not okay in `pythonPackages`. This would use Python 3.10 for `python311Packages.dashing`. Fixed in https://github.com/NixOS/nixpkgs/pull/244135

Reply:
Thank you for pointing that out :)